recipe/meta.yaml

```diff
@@ -13,28 +13,28 @@ source:
 
 build:
   number: 0
+  noarch: python
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
-    - toolchain
     - python
-    - cython
-    - setuptools
-    - setuptools_scm
+    - setuptools >18.0
+    - setuptools_scm >1.5.4
   run:
     - python
-    - numpy
+    - asciitree
+    - numpy >=1.7
     - fasteners
+    - numcodecs >=0.5.2
```

Comment:
Should we bump this to >=0.5.3 to guarantee no nose dependency?

Reply:
It's a good idea. Was just following up on this point in Zarr proper ( https://github.com/zarr-developers/zarr/pull/237 ).

Guess the counterpoint would be this would deviate from the PyPI package's requirements. No strong feelings from me though.

Reply:
Thanks for picking up the inconsistency. How about if we merge zarr-developers/zarr#237, then make a 2.2.0rc2 release to PyPI, then skip this PR ahead to 2.2.0rc2 as well, so we have all dependencies consistent between conda-forge and PyPI?

Reply:
Sure. That seems reasonable.